Cargo.toml

```diff
@@ -397,6 +396,12 @@ inherits = "release"
 lto = false
 codegen-units = 16
 
+# Used for fuzzing, LTO is ill-supported as of 2023-09 and so should not be enabled.
+[profile.fuzz]
```

Comment:
Makes snese, but this profile is not being specified in the `cargo bolero` invocation. Is cargo bolero using this profile transparently? Would be great to specify this explicitly I feel so that this divergence from the other cargo commands is clear(er).

Reply:
Yes, `cargo bolero` starting from 0.10 requires this profile. Basically, before that people would usually compile with release for performance (like we did), except LTO + sanitizers leads to all sorts of llvm / rustc bugs. So starting 0.10, trying to compile with `cargo bolero` leads to an error. See also https://github.com/camshaft/bolero/pull/163

I’m not sure what you mean by "specify this explicitly"?

Reply:
> I’m not sure what you mean by "specify this explicitly"?

Ah, well cargo commands usually have the `--profile` argument that you can pass in. I was thinking of something like `cargo bolero --profile=fuzz` just so that this behaviour of it is obvious, but this doesn’t seem super critical either.

Reply:
Makes sense; I’ve just added the explicitly-listed `--profile fuzz` argument :)